p/themes/Origine/origine.css

```diff
@@ -465,7 +459,7 @@ a.btn {
 	display: block;
 	color: #333;
 	font-style: italic;
-	line-height: 3em;
+	line-height: 3;
```

Comment:
While fairly unlikely to be actively harmful, that's still the wrong way to go about this as a matter of principle. Use something like padding-top/bottom .9em instead.

Reply:
I deleted this line because it is a copy from base-theme/base.css.

Reply:
To be clear, `line-height: 3(em)` seems to be abused as a way of specifying padding-top/bottom: `(3-1.2)/2 = 0.9(r)em`.

Reply:
To clear it it will be a next step in the base.css. With this changes here in these branch I will focus on the Origine.css

Reply:
Cool, thanks. :-)